Base/res/ladybird/about-pages/settings.html

```diff
@@ -74,10 +74,19 @@
                 padding: 15px 20px;
             }
 
+            .header-level-2 {
```

Comment:
Not a very big fan of class names like "foo1" and "foo2". Dunno if that's just common in web dev, but can we call these something more descriptive? Even something like "card-inner-header" and "card-inner-body" or something.

Reply:
this isn't foo1/foo2 though, it's a second-level header...
I'll call it "inner" instead, no problem